### ISSUE TITLE
Searching for playlists by the tracks that it contains

### ISSUE THREAD

**Issue by mokodoki** [spotify/web-api#82]
This is an underestimated and powerful feature, and I personally believe that it should be included as the default search method for playlists in the desktop app but at minimum a feature of the API. What better way to find people to follow? 

You find an artist you love but really isn't very popular and then search for other playlists which contain that track. It's one of the best ways that I've ever discovered new music (this feature was a part of Rdio's streaming service). It would also help lesser known artists connect with new fans which is something that prevents many new artists from joining the service to begin with.

As an app developer, I've written several little music discovery tools. This is the one feature I've been dying to see come to fruition.

**Comment by thelinmichael** [comment]
@mokodoki, thanks for contacting us! I'm adding this as a feature request so any changes will be updated here. I agree that this would strongly improve discoverability of playlists, and through them, users with similar tastes.

**Comment by mokodoki** [comment]
I see that the latest update has a feature "Discovered On..." that seems to pull up the most popular tracks containing an artist. Does this mean we might see this feature sooner rather than later? :)

**Comment by Drew3D** [comment]
@mokodoki where is this "Discovered On" feature?

**Comment by mokodoki** [comment]
It's available on version 1.0.14. You can find it on the artist page. On the desktop, there is a new "About" menu option at the top.

**Comment by thelinmichael** [comment]
@mokodoki, I can't go into details unfortunately, but later rather than sooner.

**Comment by ckim** [comment]
Any update on this feature? I just started a free trial and am shocked that this doesn't exist already.

**Comment by thelinmichael** [comment]
Hey @ckim! Unless I'm misunderstanding you, you're referring to Spotify's mobile or desktop applications. If so, feature requests and bug reports are best submitted to [Spotify's Community board](https://community.spotify.com). This GitHub repository is dedicated to [Spotify's Web API](https://developer.spotify.com/web-api). If you however _were_ referring to the Web API, please ignore this.

**Comment by zachkirsch** [comment]
Any update on adding 'search playlist by tracks' to the web API?

**Comment by Mfbeeck** [comment]
Any update on this feature? It would be really great for artists to be able to find what playlists their tracks are on. Even if it's an API feature @thelinmichael

**Comment by patricktyndall** [comment]
Agreed that this would be a powerful feature of the API. looking forward to when this is available.

**Comment by lfallo1** [comment]
Any updates on this?  Though I'd assume Spotify would implement a "playlists with song" feature into their app long before they gave 3rd party developers the capability.

**Comment by Skaifox** [comment]
@thelinmichael Any update on this feature in the API?

**Comment by marcospgp** [comment]
Any updates?

**Comment by gloscott** [comment]
Any updates? Would love to get this via API

**Comment by niklasR** [comment]
Another +1 from me, even though it doesn't looks like there is much work on this being done..

It'd be awesome to even infer user's opinions, suitabilities and genres for a given track!

**Comment by ScottMarcus** [comment]
This seems like a valuable API to have

**Comment by atasker** [comment]
Agreed that this is an important capability..

**Comment by dannyb2100** [comment]
Would like this also, seems highly valuable.

**Comment by AlanPipino** [comment]
Any updates? I'd love to have this

**Comment by yozzozo** [comment]
Hi all, Spotify won’t implement this feature due to business reasons (it would compete with their own editorial / curated playlists).  Given this, it’s extremely unlikely this feature will ever be added to either the API or apps.  @thelinmichael, feel free to set the record straight if I’m wrong.

**Comment by prossm** [comment]
Anybody know of other Github repos that have a hack for this? Maybe a Python script that could run in a straightforward way from the console? I found this Emacs controller—trying it now—but I confess the installation is a bit more complex than I expected:

https://github.com/danielfm/spotify.el
